drivers/gles2/rasterizer_scene_gles2.cpp

```diff
@@ -3300,7 +3300,7 @@ void RasterizerSceneGLES2::render_scene(const Transform &p_cam_transform, const
 		reflection_probe_count = 0;
 	}
 
-	if (env && env->bg_mode == VS::ENV_BG_CANVAS) {
+	if (storage->frame.current_rt && env && env->bg_mode == VS::ENV_BG_CANVAS) {
```

Comment:
Why is this needed here and not before? There's already code a few lines above that dereferences this pointer.